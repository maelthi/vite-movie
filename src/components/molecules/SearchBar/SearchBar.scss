@import "src/assets/scss/variables.scss";

.search {
  height: rem(80px);
}
