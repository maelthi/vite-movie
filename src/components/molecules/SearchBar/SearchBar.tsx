import "./SearchBar.scss"

const SearchBar = () => {
  return <section></section>
}

export default SearchBar
