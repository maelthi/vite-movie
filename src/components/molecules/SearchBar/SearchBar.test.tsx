import { render } from "@testing-library/react"
import SearchBar from "./SearchBar"

describe("SearchBar", () => {
  const defaultProps = {
    onInputChange: jest.fn(),
    inputValue: "Paul Newman",
    onFocusChange: jest.fn(),
  }
  it("should render component", () => {
    const { container } = render(<SearchBar {...defaultProps} />)
    expect(container).toMatchSnapshot()
  })
})
