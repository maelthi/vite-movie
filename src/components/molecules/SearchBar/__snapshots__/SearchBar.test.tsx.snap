// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should render component 1`] = `
<div>
  <div
    class="search"
  >
    <input
      class="search__input"
      placeholder="Rechercher"
      type="text"
      value="Paul Newman"
    />
  </div>
</div>
`;
